spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc

```diff
@@ -2,112 +2,125 @@
-in the Spring Cloud Task Github repository.  The documentation for
+in the Spring Cloud Task Github repository. The documentation for
 link:https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-services.html[Accessing Services with Diego SSH]
 and this link:http://pivotaljourney.blogspot.com/2016/05/connecting-gui-tool-to-mysql-service-in.html[blog entry]
-for connecting a GUI tools to the MySQL Service in PCF should help you to update the schema.
+for connecting a GUI tool to the MySQL Service in PCF should help you to update the schema.
 
 == Running Task Applications
 
 Running a task application within Spring Cloud Data Flow goes through a slightly different
 lifecycle than running a stream application. Both types of applications need to be registered
-with the appropriate artifact coordinates. Both need a definition created via the SCDF DSL.
-However, that's where the similarities end.
+with the appropriate artifact coordinates. Both need a definition created with the SCDF DSL.
+However, the similarities end there.
 
-With stream based applications, you "deploy" them with the intent that they run until they
+With stream-based applications, you "`deploy`" them with the intent that they run until they
 are undeployed. A stream definition is only deployed once (it can be scaled, but only
-deployed as one instance of the stream as a whole). However, tasks are _launched_. A single
-task definition can be launched many times. With each launch, they will start, execute,
-and shut down with PCF cleaning up the resources once the shutdown has occurred. The
+deployed as one instance of the stream as a whole). However, tasks are "`launched`". A single
+task definition can be launched many times. With each launch, the task starts, runs,
+and shuts down, with PCF cleaning up the resources once the shutdown has occurred. The
 following sections outline the process of creating, launching, destroying, and viewing tasks.
 
-=== Create a Task
+=== Creating a Task
 
-Similar to streams, creating a task application is done via the SCDF DSL or through the
-dashboard. To create a task definition in SCDF, you've to either develop a task
+Similar to streams, creating a task application is done by using the SCDF DSL or through the
+dashboard. To create a task definition in SCDF, you must either develop a task
 application or use one of the out-of-the-box link:http://docs.spring.io/spring-cloud-task-app-starters/docs/{sct-starters-core-version}/reference/htmlsingle[task app-starters].
 The maven coordinates of the task application should be registered in SCDF. For more
-details on how to register task applications, review <<_registering_a_task_application,register task applications>>
-section from the core docs.
+details on how to register task applications, see link:https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-register-task-apps[Registering a Task Application]
```

Comment:
It'd be ideal to not have `current` path in the URL. We have a version as parametrized includes; we can apply the same logic here, so we can be confident the URL won't break.